integration-tests/tests/singlestore/singlestore-common.ts

```diff
@@ -163,7 +169,7 @@ const usersMySchemaTable = mySchema.table('userstest', {
 	id: serial('id').primaryKey(),
 	name: text('name').notNull(),
 	verified: boolean('verified').notNull().default(false),
-	jsonb: json('jsonb').$type<string[]>(),
```

Comment:
We don't need to remove json type from tests here. Blob should be tested separately. You can check MySQL varbinary as long as SingleStore blob should be pretty the same as MySQL varbinary: https://docs.singlestore.com/cloud/reference/sql-reference/data-types/blob-types/